libs/partners/upstage/langchain_upstage/embeddings.py

```diff
@@ -193,16 +197,19 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
         Returns:
             List of embeddings, one for each text.
         """
-        embeddings = []
+        assert (
+            len(texts) <= MAX_EMBED_BATCH_SIZE
+        ), f"The batch should not be larger than {MAX_EMBED_BATCH_SIZE}."
```

Comment:
for most other integrations, we handle this batching (i.e. split it into multiple batches and return them all as if it were one request)

Reply:
The original intent was to ensure that self.embed_batch_size does not exceed MAX_EMBED_BATCH_SIZE.

Reply:
Is it better to manage with Field?

Reply:
Ah makes sense - I misunderstood! This looks good